src/main/java/me/contaria/seedqueue/mixin/compat/atum/OptionsScreenMixin.java

```diff
@@ -0,0 +1,72 @@
+package me.contaria.seedqueue.mixin.compat.atum;
+
+import com.bawnorton.mixinsquared.TargetHandler;
+import com.llamalad7.mixinextras.injector.wrapoperation.Operation;
+import com.llamalad7.mixinextras.injector.wrapoperation.WrapOperation;
+import me.contaria.seedqueue.SeedQueue;
+import net.minecraft.client.gui.screen.Screen;
+import net.minecraft.client.gui.screen.options.OptionsScreen;
+import net.minecraft.client.gui.widget.ButtonWidget;
+import net.minecraft.text.Text;
+import net.minecraft.text.TranslatableText;
+import org.spongepowered.asm.mixin.Dynamic;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Unique;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+@Mixin(value = OptionsScreen.class, priority = 1500)
+public abstract class OptionsScreenMixin extends Screen {
+    @Unique
+    private ButtonWidget atumButton;
+
+    protected OptionsScreenMixin(Text title) {
+        super(title);
+    }
+
+    @Unique
+    private boolean shouldClearQueue() {
+        return Screen.hasShiftDown() && SeedQueue.isActive();
+    }
+
+    @Dynamic
+    @TargetHandler(
+            mixin = "me.voidxwalker.autoreset.mixin.gui.OptionsScreenMixin",
+            name = "addStopResetsButton"
+    )
+    @WrapOperation(
+            method = "@MixinSquared:Handler",
+            at = @At(
+                    value = "NEW",
+                    target = "(IIIILnet/minecraft/text/Text;Lnet/minecraft/client/gui/widget/ButtonWidget$PressAction;)Lnet/minecraft/client/gui/widget/ButtonWidget;"
+            )
+    )
+    private ButtonWidget stopQueueWhenShiftClicked(int x, int y, int width, int height, Text text, ButtonWidget.PressAction action, Operation<ButtonWidget> original) {
+        this.atumButton = new ButtonWidget(x, y, width, height, text, button -> {
```

Comment:
Code: instead of using new ButtonWidget(...), call original.call(...) with the args
```suggestion
        this.atumButton = original.call(x, y, width, height, text, button -> {
```